v1.0/openapi3/parameters/FSPIOP-URI.yaml

```diff
@@ -3,4 +3,8 @@ in: header
 schema:
   type: string
 required: false
-description: 'The `FSPIOP-URI` header field is a non-HTTP standard field used by the API for signature verification, should contain the service URI. Required if signature verification is used, for more information, see [the API Signature document](https://github.com/mojaloop/docs/tree/master/Specification%20Document%20Set).'
+description: |
+  The `FSPIOP-URI` header field is a non-HTTP standard field used by the API
+  for signature verification, should contain the service URI. Required if
+  signature verification is used, for more information, see
+  [the API Signature document](https://github.com/mojaloop/docs/tree/master/Specification%20Document%20Set).'
```

Comment:
```suggestion
  [the API Signature document](https://github.com/mojaloop/docs/tree/master/Specification%20Document%20Set).
```

Reply:
:) good catch. thought I got all of them.